scopesim/effects/surface_list.py

```diff
@@ -21,11 +23,18 @@ def __init__(self, **kwargs):
         self.meta.update(params)
         self.meta.update(kwargs)
 
-        tbl = from_currsys(self.table, self.cmds)
-        self.surfaces = rad_utils.make_surface_dict_from_table(tbl)
```

Comment:
It seems that `make_surface_dict_from_table()` is now redundant and perhaps can be deleted?

Reply:
Yes it can. I did do that but get my branches confused, and they forgot to re-delete it from this branch. Thanks 👍

Reply:
The same can be done to `make_surface_from_row`